app/_layouts/extension.html

```diff
@@ -178,19 +241,44 @@
   {% if config_required_fields_yaml == 'config: ' %}config:
     <optional_parameter>: <value>{% else %}{{ config_required_fields_yaml }}{% endif %}
 ```
+
+`<service>` is the `id` or `name` of the Service that this plugin
+  configuration will target.
+
+{% endnavtab %}
+{% endunless %}
+{% unless page.params.konnect_examples == false %}
+{% navtab Konnect %}
+Configure this plugin on a Service:
+
+1. In Konnect, select the service on the ServiceHub page.
+2. Scroll down to **Versions** and select the version.
+3. Scroll down to **Plugins** and click **New Plugin**.
+4. Find and select the **{{ page.name }}** plugin.
+
+    <div class="alert alert-ee blue">
+    <strong>Note:</strong> If the plugin is greyed out, then it is not available
+    for your product tier. See <a href="">Kong Gateway tiers</a>.
+    </div>
+
+  {% if config_required_fields_konnect == empty %}
+5. Click **Create** to save.
+  {% else %}  
+5. Enter the following required [parameters](#parameters), updating the default/sample values as needed:
+    {{ config_required_fields_konnect }}
+6. Click **Create** to save.
+  {% endif %}
 {% endnavtab %}
 {% endunless %}
 {% endnavtabs %}
 {% endif %}
 
-`<service>` is the `id` or `name` of the Service that this plugin
-  configuration will target.
-
 {% if page.params.api_id %}
-  > **Note:** The legacy API entity has been **deprecated** in favor of Services
+<div class="alert alert-ee blue"><strong>Note:</strong>The legacy API entity has been **deprecated** in favor of Services
```

Comment:
Do we even want this here? Murillo has been requesting its removal in my plugin PRs. EE 0.32 is EOL version. Not sure customers know what "EE" is.

Reply:
I have removed it.